src/com/sbezboro/standardplugin/listeners/CreatureSpawnListener.java

```diff
@@ -23,7 +23,9 @@
 		add(EntityType.CHICKEN);
 		add(EntityType.PIG);
 		add(EntityType.SHEEP);
-		add(EntityType.VILLAGER);
```

Comment:
Villager supposed to be removed?

Reply:
yes, makes more sense to not include villagers